src/views/ChartPage.vue

```diff
@@ -0,0 +1,463 @@
+<template>
+  <div :class="this.$store.state.css">
+    <div class="chartPage menuLayout row col-12 offset-0 col-sm-12 offset-sm-0 col-md-10 offset-md-1 col-lg-12 offset-lg-0 col-xl-10 offset-xl-1 col-xxl-8 offset-xxl-2">
+      <div class="header fixed title">
+        <select v-model="currentGraph" class="selectGraph">
+          <option value="pie">{{ $t('PIE_CHART') }}</option>
+          <option value="bar">{{ $t('BAR_CHART') }}</option>
+        </select>
+      </div>
+      <div class="placeholder top">
+        <select v-model="currentGraph" class="selectGraph">
+          <option value="pie">{{ $t('PIE_CHART') }}</option>
+          <option value="bar">{{ $t('BAR_CHART') }}</option>
+        </select>
+      </div>
+
+      <div class="row dateNav mobileVersion">
+        <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-left" v-on:click="this.goToLastMonth()"/></div>
+        <div class="col-8 date-label">
+            <p class="title">{{ $d(this.getMonthAsDate(budgetMonth), 'monthString') }} <span v-if="!this.isThisYear"> {{ $d(this.getMonthAsDate(budgetMonth), 'year') }}</span></p>
+        </div>
+        <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-right" v-on:click="this.goToNextMonth()"/></div>
+      </div>
+
+      <div class="draw col-lg-7">
+        <PieChart :chartData="pieChartData" v-if="currentGraph == 'pie'"/>
+        <BarChart :chartData="chartData" v-if="currentGraph == 'bar'"/>
+      </div>
+
+      <div class="computerVersion col-lg-5">
+        <div class="row dateNav">
+          <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-left" v-on:click="this.goToLastMonth()"/></div>
+          <div class="col-8 date-label">
+              <p class="title">{{ $d(this.getMonthAsDate(budgetMonth), 'monthString') }} <span v-if="!this.isThisYear"> {{ $d(this.getMonthAsDate(budgetMonth), 'year') }}</span></p>
+          </div>
+          <div class="col-2 d-flex justify-content-center" ><button type="button" class="btn fas fa-chevron-right" v-on:click="this.goToNextMonth()"/></div>
+        </div>
+        <div v-if="currentGraph == 'pie'" class="radioSelect typeSelection">
```

Comment:
Could this be extracted as a component ? It could maybe be reused for both radio and checkbox with settings in the prop

Reply:
done